### PR TITLE
Devnet adjustments

### PR DESCRIPTION
**Pull request by lgalabru**
This PR is upgrading to a version of Rocket (currently in RC) that does not require rust nightly, to address https://github.com/hirosystems/clarinet/issues/97 and fix distribution via brew.
It's also fixing some compatibility issues with Linux, raised in https://github.com/hirosystems/clarinet/issues/94.

### CODE REVIEW

File: src/integrate/orchestrator.rs [hunk 1]
```diff
@@ -552,14 +552,14 @@ ignore_txs = false
...
                 port_bindings: Some(port_bindings),
-
                 binds: Some(vec![
                     format!("{}/conf:/etc/bitcoin", devnet_config.working_dir),
                     format!(
                         "{}/data/{}/bitcoin:/root/.bitcoin",
                         devnet_config.working_dir, boot_index
                     ),
                 ]),
+                extra_hosts: Some(vec!["host.docker.internal:172.17.0.1".into()]),
```
Comment (zone117x) [inline src/integrate/orchestrator.rs:562]: Last time I looked at this issue (over a year ago), I think it was possible for the docker gateway to be a different IP. Maybe needs to be detected, or configurable?

Comment (wileyj) [reply src/integrate/orchestrator.rs:562]: it is possible, that was my thought as well - hardcoding the IP here will surface issues with users that use different subnets for docker

Comment (lgalabru) [reply src/integrate/orchestrator.rs:562]: Yeah the hard coded IP address was an unsuccessful attempt to patch the issue, will revert.

**Comment by lgalabru** [comment]
Was able to successfully run devnet on a linux box, with the latest change.

**Pull request merged**